aws_quota/check/eks.py

```diff
@@ -11,4 +24,30 @@ class ClusterCountCheck(QuotaCheck):
 
     @property
     def current(self):
-        return len(self.get_client(self.service_code).list_clusters()['clusters'])
+        return len(get_all_eks_clusters(self.get_client(self.service_code)))
+
+
+class ControlPlaneSecurityGroupsPerCluster(InstanceQuotaCheck):
+    key = "eks_control_plane_sg_per_cluster"
+    description = "Control plane security groups per cluster"
+    scope = QuotaScope.INSTANCE
+    service_code = 'eks'
+    quota_code = 'L-11427A54'
+    instance_id = 'Cluster'
+    used_services = [service_code]
+
+    @staticmethod
+    def get_all_identifiers(session: boto3.Session) -> typing.List[str]:
+        return get_all_eks_clusters(get_client(session, 'eks'))
+
+    @property
+    def current(self):
+        eks = self.get_client(self.service_code).describe_cluster(name=self.instance_id)
+        # securityGroupIds:
+        # The security groups associated with the cross-account elastic network interfaces that are used to allow
+        # communication between your nodes and the Kubernetes control plane.
+        #
+        # clusterSecurityGroupId:
+        # The cluster security group that was created by Amazon EKS for the cluster. Managed node groups use this
+        # security group for control-plane-to-data-plane communication.
+        return len(eks['securityGroupIds'] + eks['clusterSecurityGroupId'])
```

Comment:
I think those are under 'cluster' and 'resourcesVpcConfig' e.g. it should be:
`eks['cluster']['resourcesVpcConfig']['securityGroupIds'] + eks['cluster']['resourcesVpcConfig']['clusterSecurityGroupId']`
with some extra checks that those keys exist because I think they are optional.

Here is an example response from the boto3 doc:
```
{
    'cluster': {
        'version': '1.10',
        'name': 'devel',
        'arn': 'arn:aws:eks:us-west-2:012345678910:cluster/devel',
        'certificateAuthority': {
            'data': 'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRFNE1EVXpNVEl6TVRFek1Wb1hEVEk0TURVeU9ESXpNVEV6TVZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTTZWCjVUaG4rdFcySm9Xa2hQMzRlVUZMNitaRXJOZGIvWVdrTmtDdWNGS2RaaXl2TjlMVmdvUmV2MjlFVFZlN1ZGbSsKUTJ3ZURyRXJiQyt0dVlibkFuN1ZLYmE3ay9hb1BHekZMdmVnb0t6b0M1N2NUdGVwZzRIazRlK2tIWHNaME10MApyb3NzcjhFM1ROeExETnNJTThGL1cwdjhsTGNCbWRPcjQyV2VuTjFHZXJnaDNSZ2wzR3JIazBnNTU0SjFWenJZCm9hTi8zODFUczlOTFF2QTBXb0xIcjBFRlZpTFdSZEoyZ3lXaC9ybDVyOFNDOHZaQXg1YW1BU0hVd01aTFpWRC8KTDBpOW4wRVM0MkpVdzQyQmxHOEdpd3NhTkJWV3lUTHZKclNhRXlDSHFtVVZaUTFDZkFXUjl0L3JleVVOVXM3TApWV1FqM3BFbk9RMitMSWJrc0RzQ0F3RUFBYU1qTUNFd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFNZ3RsQ1dIQ2U2YzVHMXl2YlFTS0Q4K2hUalkKSm1NSG56L2EvRGt0WG9YUjFVQzIrZUgzT1BZWmVjRVZZZHVaSlZCckNNQ2VWR0ZkeWdBYlNLc1FxWDg0S2RXbAp1MU5QaERDSmEyRHliN2pVMUV6VThTQjFGZUZ5ZFE3a0hNS1E1blpBRVFQOTY4S01hSGUrSm0yQ2x1UFJWbEJVCjF4WlhTS1gzTVZ0K1Q0SU1EV2d6c3JRSjVuQkRjdEtLcUZtM3pKdVVubHo5ZEpVckdscEltMjVJWXJDckxYUFgKWkUwRUtRNWEzMHhkVWNrTHRGQkQrOEtBdFdqSS9yZUZPNzM1YnBMdVoyOTBaNm42QlF3elRrS0p4cnhVc3QvOAppNGsxcnlsaUdWMm5SSjBUYjNORkczNHgrYWdzYTRoSTFPbU90TFM0TmgvRXJxT3lIUXNDc2hEQUtKUT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=',
        },
        'createdAt': 1527807879.988,
        'endpoint': 'https://A0DCCD80A04F01705DD065655C30CC3D.yl4.us-west-2.eks.amazonaws.com',
        'resourcesVpcConfig': {
            'securityGroupIds': [
                'sg-6979fe18',
            ],
            'subnetIds': [
                'subnet-6782e71e',
                'subnet-e7e761ac',
            ],
            'vpcId': 'vpc-950809ec',
        },
        'roleArn': 'arn:aws:iam::012345678910:role/eks-service-role-AWSServiceRoleForAmazonEKS-J7ONKE3BQ4PI',
        'status': 'ACTIVE',
    },
    'ResponseMetadata': {
        '...': '...',
    },
}
```

Reply:
Thanks for catching! Fixed the code. PTAL